### DISCUSSION TITLE
Remove or edit ingedrients unit

### DISCUSSION THREAD

**Discussion by Roycinger** [mealie-recipes/mealie#3209] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

After adding a unit in a recipe with "Disable Ingredient Amounts" deselected, once a unit is added, it cannot be removed from the list or edited unless editing the mealie.db manually.

### Please provide a concise description of the feature that would resolve your issue.

Implement a feature in the settings to edit the underlying DB table "ingredient_units"

### Please consider and list out some caveats or tradeoffs made in your design decision

no caveats, no tradeoffs

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Have you tried clicking the "X" button next to the unit?

![image](https://github.com/mealie-recipes/mealie/assets/71845777/9b5edca5-603d-4eb7-abdd-d2e5119163e5)

I can't reproduce this issue. Clicking on the "X" and then saving removes the unit on the demo

**Reply by Roycinger** [reply]
Thanks for getting back!
By removing I was speaking about removing the unit from the DB, not just from the recipe. 
Also, to my knowledge, the units cant be edited (i.e. changing "Gramm" to "g")

**Reply by michael-genson** [reply]
Have you checked out the data management page? You can add/edit/remove foods, units, and labels there.

Click on your profile (upper left) and scroll down to "Manage Data"

**Reply by Roycinger** [reply]
You're absolute right, thanks for guiding me in the right direction!

**Comment by Roycinger** [comment]
> Have you checked out the data management page? You can add/edit/remove foods, units, and labels there.
> 
> Click on your profile (upper left) and scroll down to "Manage Data"
